libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts

```diff
@@ -210,6 +207,8 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
     return this.#_triggerButton;
   }
 
+  public destroyRef = inject(DestroyRef);
```

Comment:
This can be `protected` instead of `public` since it's only referenced in the template.